preview/clean.js

```diff
@@ -0,0 +1,181 @@
+'use strict';
+
+const child_process = require('child_process');
+const dedent = require('dedent');
+const https = require('https');
+const os = require('os');
+const path = require('path');
+
+module.exports = Cleaner;
+module.exports.exec_git = exec_git;
```

Comment:
Might be slightly better to export an object and destructure it when requiring vs attaching `exec_git` to the Cleaner function.

```Javascript
module.exports = { Cleaner, exec_git }

// elsewhere
const { Cleaner, exec_git } = require("../clean.js")
```

But not sure how important being able to call `Cleaner.exec_git` is. ES6 modules are supported in Node.js 12 not sure if we're using that, but it would allow you to have your cake and eat it too.